cng/ecdh.go

```diff
@@ -25,14 +25,14 @@ type ecdhAlgorithm struct {
 func loadEcdh(curve string) (h ecdhAlgorithm, bits uint32, err error) {
 	var id string
 	switch curve {
-	case "P-224", "X25519":
```

Comment:
Are we changing P-224 to `errUnknownCurve` on purpose?

Reply:
Yep, upstream [crypto/ecdh](https://pkg.go.dev/crypto/ecdh@master#pkg-index) does not implement P-224, so we shouldn't treat it as unsupported.